core/deployment/src/main/java/io/quarkus/deployment/pkg/PackageConfig.java

```diff
@@ -49,6 +50,13 @@ public interface PackageConfig {
      */
     Optional<String> outputName();
 
+    /**
+     * The timestamp used as a reference for generating the packages (e.g. for the creation timestamp of ZIP entries).
+     * <p>
+     * The approach is similar to what is done by the maven-jar-plugin with `project.build.outputTimestamp`.
+     */
+    Optional<Instant> outputTimestamp();
```

Comment:
Is this similar to what I suggested before in this enhancement request (#41070)? `quarkus.build.timestamp`

Reply:
If we decide to go this way, it could be used to do what you were asking for.

For now, the main reason for it is to make sure the Quarkus jars are generated reproducibly.